Codeinwp/otter-blocks
# feat: Stripe API change in Editor

Closes #1609 .


### Summary


Instead of redirecting the user to Dashboard, allow it to change the API Key in Editor like in Google Maps.

Side note: A very nice page on resolvers https://unfoldingneurons.com/2020/wordpress-data-store-properties-resolvers

### Screenshots 

https://user-images.githubusercontent.com/17597852/237044664-d0d0b9c2-5792-42f7-8d64-3e381e7f18ee.mp4

----

### Test instructions


⚠️ Make sure to have a Stripe API Key and some test products!

Follow the steps in the video and make sure that:
- It is not stuck; if you put in a valid API Key and refresh the page, the products selection will appear or the products themselves (if they were already selected)
- You can NOT find the correct API Key. There should be NO method to see the **VALID** API Key.



---- 

### Checklist before the final review
